src/main/webapp/i18n/de/buildQueue.json

```diff
@@ -26,7 +26,22 @@
                 "status": "Status",
                 "buildLogs": "Build Logs"
             },
-            "cancelAll": "Alle abbrechen"
+            "cancelAll": "Alle abbrechen",
+            "statistics": {
+                "title": "Übersicht",
+                "titleTooltip": "Daten werden für den letzten Zeitraum abgerufen, der durch die Zeitspanne definiert ist.",
```

Comment:
I would change the translation to "Daten für den definierten Zeitraum werden abgerufen"